framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -718,6 +751,41 @@ public void clearValue(JavaExpression expr) {
         }
     }
 
+    /**
+     * Returns the abstract value for {@link Tree} {@code t}, or {@code null} if no information is
+     * available.
+     *
+     * @param tree a tree
+     * @return the abstract value for {@link Tree} {@code t}, or {@code null} if no information is
+     *     available
+     */
+    public @Nullable V getValue(Tree tree) {
```

Comment:
Why didn't we need this method before? Or is this moved from somewhere and I missed it?